bundles/org.openhab.ui/web/src/pages/settings/items/item-edit.vue

```diff
@@ -74,7 +74,7 @@ export default {
     return {
       ready: false,
       item: {},
-      items: [],
+      items: null,
```

Comment:
While this probably solves #1062 indeed, it causes a new error in the console when creating a new item (Settings > Item > Add New Item):

```
[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "items"

found in

---> <ItemForm> at src/components/item/item-form.vue
       <F7Col>
         <F7Block>
           <F7PageContent>
             <F7Page>
               <ItemEdit> at src/pages/settings/items/item-edit.vue
```

Reply:
Argh, missed that. Actually, that code line in question seems to have been able to cause these lines even before. Will adjust.